librz/type/path.c

```diff
@@ -36,74 +36,169 @@ RZ_API void rz_type_path_free(RZ_NULLABLE RzTypePath *tpath) {
 	free(tpath);
 }
 
+static RzType *path_walker_parse_bracket(const RzTypeDB *typedb, RzType *parent, const char *path, size_t *i, st64 *offset) {
+	size_t nd = 0;
+	RzType *typd = parent;
+	st64 curd_off = 1; // prod of all dim sizes
```

Comment:
Why curd? Please try to give more meaningful names. Also the same in the comment

Reply:
"current dimesion". Fixed :)